hedera-node/src/test/java/com/hedera/services/store/models/IdTest.java

```diff
@@ -67,15 +70,20 @@ void equalsDiscriminates() {
 	void conversionsWork() {
 		final var id = new Id(1, 2, 3);
 		final var entityId = new EntityId(1, 2, 3);
-		final var merkleEntityId = new MerkleEntityId(1, 2, 3);
 		final var grpcAccount = IdUtils.asAccount("1.2.3");
 		final var grpcToken = IdUtils.asToken("1.2.3");
+		final var contractId = ContractID.newBuilder().setShardNum(1).setRealmNum(2).setContractNum(3).build();
+		final var address = Address.wrap(Bytes.wrap(EntityIdUtils.asSolidityAddress(contractId)));
 
 		assertEquals(entityId, id.asEntityId());
 		assertEquals(grpcAccount, id.asGrpcAccount());
 		assertEquals(grpcToken, id.asGrpcToken());
 		assertEquals(id, Id.fromGrpcAccount(grpcAccount));
 		assertEquals(id, Id.fromGrpcToken(grpcToken));
+		assertEquals(contractId, id.asGrpcContract());
+		assertEquals(id, Id.fromGrpcContract(contractId));
+		assertEquals(id, Id.fromEvmAddress(address));
+		assertEquals(address, id.asEvmAddress());
```

Comment:
nit: Let's be consistent about the ordering:
1. assert `Id` <-> `id.asSomething()` 
2. assert `Id` <-> `Id.fromSomething(x)`